### PR TITLE
improve map performance [CPP-916]

### PR DESCRIPTION
**Pull request by adrian-kong**
https://docs.mapbox.com/help/troubleshooting/mapbox-gl-js-performance/

- disables sending metrics to mapbox
- enables optimize base styles
- enables attribute use desktop opengl
- half required points to draw ellipse (might be better to scale this w.r.t distance on map since small circles looks ok rendering ~ 8 points)

### CODE REVIEW

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4464899614)

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4465864245)

File: resources/web/map/js/trajectory_raw.js [hunk 1]
```diff
@@ -159,12 +160,13 @@ function syncLayers() {
  * @param points optional number of points to render ellipse, higher for smoothness
```
Comment (pcrumley) [inline resources/web/map/js/trajectory_raw.js:160]: you need to move this doc string to be an inline comment about the `let points = 16; line.` OR undo the refactor of the optional param to a function.

**Review by pcrumley — CHANGES_REQUESTED** [review]
This a pretty difficult thing to review other than just a rubber stamp, but it looks fine? 

marking as request changes because you need to update the documentation or undo a refactor

**Review by pcrumley — APPROVED** [review]
I kind of disagree with decision to remove the comments, i think putting it as an inline comment makes more sense but i don't think it should block merge.

**Comment by adrian-kong** [comment]
> I kind of disagree with decision to remove the comments, i think putting it as an inline comment makes more sense but i don't think it should block merge.

think it might be changed in later PRs if the optimal # of points for zoom levels is figured out
but for now might just leave default 16 pts => 20 degrees

**Pull request merged**